mailer.js

```diff
@@ -375,11 +406,13 @@ module.exports = function (log) {
       template: 'newDeviceLoginEmail',
       templateValues: {
         device: this._formatUserAgentInfo(message),
+        ip: message.ip,
+        location: gettext(this._constructLocationString(message.location)),
```

Comment:
See [this comment I made above](https://github.com/mozilla/fxa-auth-mailer/pull/180/files/3bc4a05af22f535f79e546e95b828802e1a3e7d9#r71302053) about translating the location.